azurerm/internal/services/azurestackhci/validate/cluster_name.go

```diff
@@ -0,0 +1,18 @@
+package validate
+
+import "fmt"
+
+func ClusterName(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		errors = append(errors, fmt.Errorf("expected type of %q to be string", k))
+		return
```

Comment:
Can we be consistent here to either implicit return or explicit?

Reply:
Good catch. Updated